code/modules/surgery/bodyparts/dismemberment.dm

```diff
@@ -365,20 +368,14 @@
 	if(can_be_disabled)
 		update_disabled()
 
+	synchronize_bodytypes(new_limb_owner)
 	new_limb_owner.updatehealth()
 	new_limb_owner.update_body()
-	new_limb_owner.update_hair()
 	new_limb_owner.update_damage_overlays()
-
+	return .
```

Comment:
Better question, why not just return TRUE here? runtime safety?

Reply:
It's how it was when I found it so I just didn't touch it.

Reply:
remove don't improve

Reply:
works for me